### ISSUE TITLE
online audio samples

### ISSUE THREAD

**Issue by sammlapp** [kitzeslab/opensoundscape#306]
we should make a few online audio datasets that can very easily be loaded using opensoundscape, for general testing and prototyping. For instance:

```
test_audio_set, _ = Audio.get_online_dataset('basic_audio')
test_audio_set[0].trim(0,5) # test_audio_set is a list of Audio objects
```

or 
```
test_audio_set, labels = Audio.get_online_dataset('classification_dataset')
model.train(labels) # this wouldn't quite work because you need the files on your disk to train
```

**Comment by sammlapp** [comment]
Thinking we could have sets of both .wav and .mp3 files, some with labels, in a standard place. I don't think Audio.get_online_dataset() is a good API anymore, though I do think there is some analog in sklearn where you can access standard datasets from inside the sklearn API - may be worth copying or looking at

**Comment by sammlapp** [comment]
as of 0.6.1 we provide and use some standard audio sets via box
